### PR TITLE
Require 4.3.0 bugs for master

### PR DESCRIPTION
**Pull request by eparis**
Also set up release-4.3 so it works when we branch. (Though no one
should be sending PRs there yet)

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -1903,13 +1904,14 @@ bugzilla:
...
-    # openshift-4.3:
-    #   target_release: 4.3.z
-    #   validate_by_default: true
+    openshift-4.3:
+      target_release: 4.3.z
+      validate_by_default: true
+      dependent_bug_target_release: 4.4.0
     release-4.3:
-      target_release: 4.3.0
+      target_release: 4.3.z
```
Comment (wking) [inline core-services/prow/02_config/_plugins.yaml]: We don't want this until we cut 4.3.0, right?

Comment (eparis) [reply core-services/prow/02_config/_plugins.yaml]: we wouldn't want people merging to release-4.3 at all at this point, they should be using master, so it'd be a good thing...

**Comment by stevekuznetsov** [comment]
/hold

As discussed on Slack I am not sure a blanket application to all repos with a `master` branch is what we want.

**Comment by stevekuznetsov** [comment]
Technically repos that do not use the BZ process do not need to require the valid label for merging but having the bot deem everything invalid would just be spam.

**Pull request closed without merging**